odo/backends/sql.py

```diff
@@ -172,7 +172,9 @@ def discover_typeengine(typ):
     if isinstance(typ, (sa.NUMERIC, sa.DECIMAL)):
         return datashape.Decimal(precision=typ.precision, scale=typ.scale)
     if isinstance(typ, (sa.String, sa.Unicode)):
-        return datashape.String(typ.length, typ.collation)
+        return datashape.String(typ.length, None)
```

Comment:
Ok after doing a little digging... 

the type type is NVARCHAR or NCHAR, that's by definition unicode. (I think SQL Server uses UTF-16)

If the type is CHAR or VARCHAR, then the code page (which I think is basically the same thing as encoding) is dependent on the collation. 

https://msdn.microsoft.com/en-us/library/ms190305.aspx

http://www.sqlservercentral.com/blogs/rocks/2012/01/09/revised-difference-between-collation-sql_latin1_general_cp1_ci_as-and-latin1_general_ci_as/

So right now, I'm thinking there's gonna have to be some logic that says Collation -> Code Page -> python encoding. Doesn't look fun.